### ISSUE TITLE
New L1 Anomaly (ready for finishing)

### ISSUE THREAD

**Issue by Chhkt** [MoreEventsMod/More_Events_Mod_Beta#229]
Author: Chhkt
L1 Anomaly: A Game of Catch

Image: GFX_evt_landing_ship
Can spawn on any habitable planet

Initial Text: 
The science crew has taken a special interest in some ancient ruins on the planet’s surface. (scientist name) recommends additional study.

Success: 60%
The extra time taken to “study” the ancient ruins was actually a disguise by the crew to relax and play a game of hide and seek. It wasn't a complete waste of time though, as one of the crew ran headfirst into a wall with a very complex set of hieroglyphics engraved in it.

Dialogue: “Very well.” (player gains 200 society research)

Failure: 40%
The extra time taken to “study” the ancient ruins was actually a disguise by the crew to relax and play a game hide and seek. While technically they were doing their job by exploring the ruins, it would be best to remind (Scientist name) to focus on the mission.

Dialogue: “Stay focused.”

**Comment by Elimdur** [comment]
This attitude of "Ey the air is breathable, lets take our helmets off" is what gets idiots killed in movies. The idea of finding something on accident sounds interesting but it may need some more work to get the stellaris feel.
I mean even in star trek those guys would most likely get a disciplinary action for that.

**Comment by Chhkt** [comment]
I had thought about using the "stay focused" line to add the Paranoid trait to the science officer and making the dialogue more harsh. Do you think that the success option should be fairly negative as well?

**Comment by Elimdur** [comment]
yeah it should atleast remind them of their duty instead of just letting that one go.

also maybe you should think about how to describe that relax time a bit less like the camping scene in star ship troopers ;)

**Comment by Chhkt** [comment]
Going to have to watch that on youtube, don't remember it. I was thinking of having them find a decrepit sports field left behind by some long lost race and they decided to play a game on it but that it was too farfetched.

**Comment by tzerrell** [comment]
Fun idea, but I agree this needs a bit of revision before it's ready to go live.
